pkgs/development/compilers/gcc/4.5/default.nix

```diff
@@ -256,11 +256,7 @@ stdenv.mkDerivation ({
     ;
 
   # TODO(@Ericson2314): Always pass "--target" and always prefix.
-  configurePlatforms =
-    # TODO(@Ericson2314): Figure out what's going wrong with Arm
-    if buildPlatform == hostPlatform && hostPlatform == targetPlatform && targetPlatform.isAarch32
-    then []
-    else [ "build" "host" ] ++ stdenv.lib.optional (targetPlatform != hostPlatform) "target";
+  configurePlatforms = [ "build" "host" ] ++ stdenv.lib.optional (targetPlatform != hostPlatform) "target";
```

Comment:
Is it runtime-tested that this fixes it?

Reply:
No it is not. Unless we have some CI that does this, I was hoping you might :).

Reply:
The only other issue I remember is ld picking a wrong emulation. If this also doesn't solve that we have the means to force it from nix now.

Reply:
IIRC the LD emulation thing was the actual thing that prevented it from working.

Reply:
Also as far as I know we are using the same triplets that other distros do, or maybe that was one of the things to double-check - IIRC gnu-efi had some problems that I had to work around on ARM. I can still try to build this though once some other builds finish.

Reply:
I know on cross this was definitely helps: the `xxx-unknown-linux-yyy` where `yyy` was not `gnu` usually broke it before https://git.savannah.gnu.org/cgit/config.git/commit/?id=50a0413a5ba9684aa9247444c60a5cea47c4435d

If you can test that would still be a big help, thanks! Even if I do need to fix the emulation after, it's probably easier to reproduce than dredge up the old build failure.

Reply:
Doesn't work:

````
these derivations will be built:
  /nix/store/d5zlm9d7rxwr28yhj2ngsx5sg1bimkix-gcc-7.3.0.drv
  /nix/store/iis49fizzflqj7hi4v87rvdw3kryyx83-bootstrap-stage4-gcc-wrapper-7.3.0.drv
  /nix/store/xsn2587jpmr1i2xliskmaywhgkcdfr2m-bootstrap-stage4-stdenv-linux.drv
  /nix/store/8x6rlbibw5154ibgyqs4llfhs4f6zzmk-attr-2.4.47.drv
  /nix/store/nms8kfwvkr9zzkycv2yf4788fvdzjnic-xz-5.2.3.drv
  /nix/store/vqqpz0yfgqacmp2l8kpkjbnqavv08z77-acl-2.2.52.drv
  /nix/store/582wls0hgwdw23g144ak38i54jys4xvc-coreutils-8.29.drv
  /nix/store/0wzjmgsgi339jrynh6iw5axn6glvk0l2-diffutils-3.6.drv
  /nix/store/226kn5zl2vxnaxrj5nypm2ckfjdaha35-gawk-4.2.1.drv
  /nix/store/230la4ma5dcdcvh6ind2v2zc485gb7hm-bzip2-1.0.6.0.1.drv
  /nix/store/2asxz17f2ga5yy02xs4sfbfdn1danfcq-paxctl-0.9.drv
  /nix/store/pmp2wm8qks8j7xd9rwvxs7n96ghpdy6s-gnu-config-2016-12-31.drv
  /nix/store/488wym696ld2rylj7wiw3dvmqdqj3ybi-hook.drv
  /nix/store/56mh6ipcwabhwaxbp9kr5dkh2vg516qj-patchelf-0.9.drv
  /nix/store/6cbbsdnp5z5ixllqylahmnf3mshax52n-findutils-4.6.0.drv
  /nix/store/2i6af99zyanqk7jyw8jca4sz6hgqbscb-texinfo-6.5.drv
  /nix/store/kcd9jypvwdzi24sq85g8zady0dzkadyg-lzip-1.20.drv
  /nix/store/lmh4njarq5mjcm4wzcsc39jxb9m7g86r-ed-1.14.2.drv
  /nix/store/98jf8l82z0q4mfj3kgjfggz3hhq0vs39-patch-2.7.6.drv
  /nix/store/9kjcyn273nzr9kypxvbzxbkp6hk1dwvq-pcre-8.41.drv
  /nix/store/b45cwrilkcqvcz2wwc2zbi76n1g4ahzf-gnutar-1.30.drv
  /nix/store/gycxxwkzdb1mpng6qvkjjlhnrn3sj51m-binutils-2.30.drv
  /nix/store/hwbrcbxinffnaarph1ann9fnm60nma1y-bash-4.4-p19.drv
  /nix/store/ixsc8ijq1ipgqdqvg5l69wjc0b3gdl3z-gnused-4.4.drv
  /nix/store/pcgg65gs3l70al89p11pr1v8mkjz049p-gzip-1.9.drv
  /nix/store/z6y2xahm6sf0likyi5mbmjw6wvh2h8jl-gnugrep-3.1.drv
  /nix/store/r27yz04qid20wysgz4bq5w3f74kfj13v-binutils-wrapper-2.30.drv
  /nix/store/qf76yb3j7y7csylc2p1414jv2n0xg6zi-gcc-wrapper-7.3.0.drv
  /nix/store/wr93q0wxcbx8snpif3c2h3jv0lq16saz-gnumake-4.2.1.drv
  /nix/store/10g0yyvv1zvlb25qqzq37ig7ppjpw69g-stdenv-linux.drv
  /nix/store/qipy7yy9z9jfpx0nfnjnycji1krgpmmv-hello-2.10.drv
building '/nix/store/d5zlm9d7rxwr28yhj2ngsx5sg1bimkix-gcc-7.3.0.drv'...
$CPATH is `/nix/store/3220yz4q0b2hjbxqglx5g1x65svgl7p9-zlib-1.2.11-dev/include'
$LIBRARY_PATH is `/nix/store/sx6gcnlkv79giirvvnp0f4kic1wfqnmz-zlib-1.2.11/lib'
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/m16cs9a00znx11mqfbkmk5f3i7gyn1ca-gcc-7.3.0.tar.xz
source root is gcc-7.3.0
setting SOURCE_DATE_EPOCH to timestamp 1516870173 of file gcc-7.3.0/MD5SUMS
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
applying patch /nix/store/s6lavkpjqzihfs0vjfnnz7lizpsfmgnl-riscv-pthread-reentrant.patch
patching file gcc/config/riscv/linux.h
applying patch /nix/store/6m27y27zvzsjn1ir4y8mm9nc9xnh2sgx-riscv-no-relax.patch
patching file gcc/config/riscv/riscv.c
Hunk #1 succeeded at 3676 (offset -303 lines).
patching file gcc/config/riscv/riscv.opt
Hunk #1 succeeded at 102 (offset -4 lines).
patching file gcc/doc/invoke.texi
Hunk #1 succeeded at 976 (offset -66 lines).
Hunk #2 succeeded at 20976 (offset -2127 lines).
applying patch /nix/store/pz03lhak99nzyyni72m14zsjzfjvqvr8-no-sys-dirs.patch
patching file gcc/cppdefault.c
patching file gcc/gcc.c
Hunk #1 succeeded at 1464 (offset 302 lines).
patching script interpreter paths in ./fixincludes/configure
./fixincludes/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libcpp/configure
./libcpp/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libcilkrts/configure
patching script interpreter paths in ./libssp/configure
./libssp/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libffi/configure
./libffi/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libada/configure
./libada/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libhsail-rt/configure
./libhsail-rt/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libgfortran/configure
./libgfortran/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libvtv/configure
./libvtv/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libobjc/configure
./libobjc/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libgo/configure
./libgo/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./gcc/configure
./gcc/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libdecnumber/configure
./libdecnumber/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libgcc/configure
patching script interpreter paths in ./configure
./configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./gnattools/configure
./gnattools/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libbacktrace/configure
./libbacktrace/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libatomic/configure
./libatomic/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./lto-plugin/configure
./lto-plugin/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libiberty/configure
./libiberty/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./intl/configure
./intl/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./zlib/configure
./zlib/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libstdc++-v3/configure
./libstdc++-v3/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libsanitizer/configure
./libsanitizer/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libcc1/configure
./libcc1/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libquadmath/configure
./libquadmath/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libmpx/configure
patching script interpreter paths in ./liboffloadmic/plugin/configure
patching script interpreter paths in ./liboffloadmic/configure
patching script interpreter paths in ./libgomp/configure
./libgomp/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./gotools/configure
./gotools/configure: interpreter directive changed from " /bin/sh" to "/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sh"
patching script interpreter paths in ./libitm/configure
fixing the `GLIBC_DYNAMIC_LINKER', `UCLIBC_DYNAMIC_LINKER', and `MUSL_DYNAMIC_LINKER' macros...
  fixing `gcc/config/kfreebsd-gnu.h'...
  fixing `gcc/config/kopensolaris-gnu.h'...
  fixing `gcc/config/aarch64/aarch64-freebsd.h'...
  fixing `gcc/config/aarch64/aarch64-linux.h'...
  fixing `gcc/config/alpha/freebsd.h'...
  fixing `gcc/config/alpha/linux-elf.h'...
  fixing `gcc/config/arc/linux.h'...
  fixing `gcc/config/arm/freebsd.h'...
  fixing `gcc/config/arm/linux-eabi.h'...
  fixing `gcc/config/arm/linux-elf.h'...
  fixing `gcc/config/c6x/uclinux-elf.h'...
  fixing `gcc/config/cris/linux.h'...
  fixing `gcc/config/frv/linux.h'...
  fixing `gcc/config/i386/dragonfly.h'...
  fixing `gcc/config/i386/freebsd.h'...
  fixing `gcc/config/i386/freebsd64.h'...
  fixing `gcc/config/i386/gnu-user.h'...
  fixing `gcc/config/i386/gnu-user64.h'...
  fixing `gcc/config/i386/gnu.h'...
  fixing `gcc/config/i386/kfreebsd-gnu.h'...
  fixing `gcc/config/i386/kfreebsd-gnu64.h'...
  fixing `gcc/config/i386/linux.h'...
  fixing `gcc/config/i386/linux64.h'...
  fixing `gcc/config/ia64/freebsd.h'...
  fixing `gcc/config/ia64/linux.h'...
  fixing `gcc/config/m32r/linux.h'...
  fixing `gcc/config/m68k/linux.h'...
  fixing `gcc/config/microblaze/linux.h'...
  fixing `gcc/config/mips/gnu-user.h'...
  fixing `gcc/config/mips/linux.h'...
  fixing `gcc/config/mn10300/linux.h'...
  fixing `gcc/config/nios2/linux.h'...
  fixing `gcc/config/pa/pa-linux.h'...
  fixing `gcc/config/riscv/linux.h'...
  fixing `gcc/config/rs6000/freebsd64.h'...
  fixing `gcc/config/rs6000/linux.h'...
  fixing `gcc/config/rs6000/linux64.h'...
  fixing `gcc/config/rs6000/sysv4.h'...
  fixing `gcc/config/rs6000/sysv4le.h'...
  fixing `gcc/config/s390/linux.h'...
  fixing `gcc/config/sh/linux.h'...
  fixing `gcc/config/sparc/freebsd.h'...
  fixing `gcc/config/sparc/linux.h'...
  fixing `gcc/config/sparc/linux64.h'...
  fixing `gcc/config/xtensa/linux.h'...
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
updateAutotoolsGnuConfigScriptsPhase
Updating Autotools / GNU config script to a newer upstream version: ./config.sub
Updating Autotools / GNU config script to a newer upstream version: ./config.guess
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
configure flags: --prefix=/nix/store/3j52fq8qdrsfa60dn3ars8gk065bgpv0-gcc-7.3.0 --with-gmp-include=/nix/store/5hxhfhcxq8skyx1rzifhk4q42dnpkxby-gmp-6.1.2-dev/include --with-gmp-lib=/nix/store/648gmmmcpnfg38a2x3dibnqrbb37kpnp-gmp-6.1.2/lib --with-mpfr-include=/nix/store/ghwny84f85zj2fbsmzyph9lg6hq5a2q1-mpfr-3.1.6-dev/include --with-mpfr-lib=/nix/store/xcb3mlimgih8gpny24974z0zhmi1lq22-mpfr-3.1.6/lib --with-mpc=/nix/store/6c4v8bz9s4lhp21xi0m73ldgwvxdckal-libmpc-1.1.0 --with-libelf=/nix/store/jhppd35iwmsl5hsn1v8r0xxwh16w6r81-libelf-0.8.13 --with-native-system-header-dir=/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include --enable-lto --disable-libstdcxx-pch --without-included-gettext --with-system-zlib --enable-static --enable-languages=c\,c++ --disable-multilib --enable-plugin --with-isl=/nix/store/nqsz7d52k99h9kym94ja1lhm5vfx0rmn-isl-0.17.1 --with-arch=armv7-a --with-fpu=vfpv3-d16 --with-float=hard --build=armv7l-unknown-linux-gnu --host=armv7l-unknown-linux-gnu
checking build system type... armv7l-unknown-linux-gnu
checking host system type... armv7l-unknown-linux-gnu
checking target system type... armv7l-unknown-linux-gnu
checking for a BSD-compatible install... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/install -c
checking whether ln works... yes
checking whether ln -s works... yes
checking for a sed that does not truncate output... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sed
checking for gawk... gawk
checking for libatomic support... yes
checking for libcilkrts support... no
checking for libitm support... yes
checking for libsanitizer support... yes
checking for libvtv support... yes
checking for libmpx support... no
checking for libhsail-rt support... no
checking for armv7l-unknown-linux-gnu-gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking whether g++ accepts -static-libstdc++ -static-libgcc... no
checking for armv7l-unknown-linux-gnu-gnatbind... no
checking for gnatbind... no
checking for armv7l-unknown-linux-gnu-gnatmake... no
checking for gnatmake... no
checking whether compiler driver understands Ada... no
checking how to compare bootstrapped objects... cmp --ignore-initial=16 $$f1 $$f2
checking for objdir... .libs
checking for the correct version of gmp.h... yes
checking for the correct version of mpfr.h... yes
checking for the correct version of mpc.h... yes
checking for the correct version of the gmp/mpfr/mpc libraries... yes
checking for isl 0.15 or later... yes
*** This configuration is not supported in the following subdirectories:
     target-libcilkrts target-libmpx gnattools gotools target-libada target-libhsail-rt target-libgfortran target-libbacktrace target-libgo target-libffi target-libobjc target-liboffloadmic
    (Any other directories should still work fine.)
checking for default BUILD_CONFIG... bootstrap-debug
checking for --enable-vtable-verify... no
checking for bison... no
checking for byacc... no
checking for yacc... no
checking for bison... no
checking for gm4... no
checking for gnum4... no
checking for m4... no
checking for flex... no
checking for lex... no
checking for flex... no
checking for makeinfo... makeinfo
checking for expect... no
checking for runtest... no
checking for ar... (cached) ar
checking for armv7l-unknown-linux-gnu-ar... (cached) ar
checking for as... (cached) as
checking for armv7l-unknown-linux-gnu-as... (cached) as
checking for armv7l-unknown-linux-gnu-dlltool... no
checking for dlltool... dlltool
checking for ld... (cached) ld
checking for armv7l-unknown-linux-gnu-ld... (cached) ld
checking for armv7l-unknown-linux-gnu-lipo... no
checking for lipo... no
checking for nm... (cached) nm
checking for armv7l-unknown-linux-gnu-nm... (cached) nm
checking for ranlib... (cached) ranlib
checking for armv7l-unknown-linux-gnu-ranlib... (cached) ranlib
checking for strip... (cached) strip
checking for armv7l-unknown-linux-gnu-strip... (cached) strip
checking for windres... (cached) windres
checking for armv7l-unknown-linux-gnu-windres... (cached) windres
checking for armv7l-unknown-linux-gnu-windmc... no
checking for windmc... windmc
checking for objcopy... (cached) objcopy
checking for armv7l-unknown-linux-gnu-objcopy... (cached) objcopy
checking for objdump... (cached) objdump
checking for armv7l-unknown-linux-gnu-objdump... (cached) objdump
checking for readelf... (cached) readelf
checking for armv7l-unknown-linux-gnu-readelf... (cached) readelf
checking for cc... cc
checking for c++... c++
checking for gcc... gcc
checking for gfortran... no
checking for gccgo... no
checking for ar... no
checking for ar... ar
checking for as... no
checking for as... as
checking for dlltool... no
checking for dlltool... dlltool
checking for ld... no
checking for ld... ld
checking for lipo... no
checking for lipo... no
checking for nm... no
checking for nm... nm
checking for objcopy... no
checking for objcopy... objcopy
checking for objdump... no
checking for objdump... objdump
checking for ranlib... no
checking for ranlib... ranlib
checking for readelf... no
checking for readelf... readelf
checking for strip... no
checking for strip... strip
checking for windres... no
checking for windres... windres
checking for windmc... no
checking for windmc... windmc
checking where to find the target ar... host tool
checking where to find the target as... host tool
checking where to find the target cc... just compiled
checking where to find the target c++... just compiled
checking where to find the target c++ for libstdc++... just compiled
checking where to find the target dlltool... host tool
checking where to find the target gcc... just compiled
checking where to find the target gfortran... host tool
checking where to find the target gccgo... host tool
checking where to find the target ld... host tool
checking where to find the target lipo... host tool
checking where to find the target nm... host tool
checking where to find the target objcopy... host tool
checking where to find the target objdump... host tool
checking where to find the target ranlib... host tool
checking where to find the target readelf... host tool
checking where to find the target strip... host tool
checking where to find the target windres... host tool
checking where to find the target windmc... host tool
checking whether to enable maintainer-specific portions of Makefiles... no
configure: creating ./config.status
config.status: creating Makefile
@nix { "action": "setPhase", "phase": "buildPhase" }
building
build flags: -j1 -l1 SHELL=/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/bash BUILD_SYSTEM_HEADER_DIR=/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include SYSTEM_HEADER_DIR=/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include NATIVE_SYSTEM_HEADER_DIR=/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include LDFLAGS_FOR_BUILD=\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-rpath\ -Wl\,/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib LDFLAGS_FOR_TARGET=-Wl\,-rpath\,/nix/store/3n43kc3ylwq1zvay8zikbq77f5p747bd-gcc-7.3.0-lib/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-rpath\ -Wl\,/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib CFLAGS_FOR_BUILD=-O2\ -I/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -B/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib/\ -idirafter\ /nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -idirafter\ /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/lib/gcc/arm-linux-gnueabihf/5.4.0/include-fixed\ \ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-rpath\ -Wl\,/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib CXXFLAGS_FOR_BUILD=-O2\ -I/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -B/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib/\ -idirafter\ /nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -idirafter\ /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/lib/gcc/arm-linux-gnueabihf/5.4.0/include-fixed\ \ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-rpath\ -Wl\,/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib FLAGS_FOR_BUILD=-O2\ -I/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -B/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib/\ -idirafter\ /nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -idirafter\ /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/lib/gcc/arm-linux-gnueabihf/5.4.0/include-fixed\ \ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-rpath\ -Wl\,/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib CFLAGS_FOR_TARGET=-O2\ -I/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -B/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib/\ -idirafter\ /nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -idirafter\ /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/lib/gcc/arm-linux-gnueabihf/5.4.0/include-fixed\ -Wl\,-rpath\,/nix/store/3n43kc3ylwq1zvay8zikbq77f5p747bd-gcc-7.3.0-lib/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-rpath\ -Wl\,/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib CXXFLAGS_FOR_TARGET=-O2\ -I/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -B/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib/\ -idirafter\ /nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -idirafter\ /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/lib/gcc/arm-linux-gnueabihf/5.4.0/include-fixed\ -Wl\,-rpath\,/nix/store/3n43kc3ylwq1zvay8zikbq77f5p747bd-gcc-7.3.0-lib/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-rpath\ -Wl\,/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib FLAGS_FOR_TARGET=-O2\ -I/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -B/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib/\ -idirafter\ /nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -idirafter\ /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/lib/gcc/arm-linux-gnueabihf/5.4.0/include-fixed\ -Wl\,-rpath\,/nix/store/3n43kc3ylwq1zvay8zikbq77f5p747bd-gcc-7.3.0-lib/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-rpath\ -Wl\,/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib BOOT_CFLAGS=-O2\ -I/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -B/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib/\ -idirafter\ /nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -idirafter\ /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/lib/gcc/arm-linux-gnueabihf/5.4.0/include-fixed\ -Wl\,-rpath\,/nix/store/3n43kc3ylwq1zvay8zikbq77f5p747bd-gcc-7.3.0-lib/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-rpath\ -Wl\,/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib BOOT_LDFLAGS=-O2\ -I/nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -B/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib/\ -idirafter\ /nix/store/mvjr671h3dwk0pc0liad672wga6j0gd8-glibc-2.27-dev/include\ -idirafter\ /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/lib/gcc/arm-linux-gnueabihf/5.4.0/include-fixed\ -Wl\,-rpath\,/nix/store/3n43kc3ylwq1zvay8zikbq77f5p747bd-gcc-7.3.0-lib/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-rpath\ -Wl\,/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib\ -Wl\,-L/nix/store/09yw4h21jjfqnx952nip1x2ajybr5zhd-glibc-2.27/lib LIMITS_H_TEST=true bootstrap
echo stage3 > stage_final
make[1]: Entering directory '/build/build'
make[2]: Entering directory '/build/build'
make[2]: Leaving directory '/build/build'
make[2]: Entering directory '/build/build'
Configuring stage 1 in ./intl
configure: creating cache ./config.cache
checking whether make sets $(MAKE)... yes
checking for a BSD-compatible install... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/install -c
checking whether NLS is requested... yes
checking for msgfmt... /nix/store/n4m8gs2s3ywd2li1z6viqrirl42lrfq6-gettext-0.19.8/bin/msgfmt
checking for gmsgfmt... /nix/store/n4m8gs2s3ywd2li1z6viqrirl42lrfq6-gettext-0.19.8/bin/msgfmt
checking for xgettext... /nix/store/n4m8gs2s3ywd2li1z6viqrirl42lrfq6-gettext-0.19.8/bin/xgettext
checking for msgmerge... /nix/store/n4m8gs2s3ywd2li1z6viqrirl42lrfq6-gettext-0.19.8/bin/msgmerge
checking for armv7l-unknown-linux-gnu-gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking build system type... armv7l-unknown-linux-gnu
checking host system type... armv7l-unknown-linux-gnu
checking for armv7l-unknown-linux-gnu-ranlib... ranlib
checking for library containing strerror... none required
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/grep
checking for egrep... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/grep -E
checking for ANSI C header files... yes
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for off_t... yes
checking for size_t... yes
checking for working alloca.h... yes
checking for alloca... yes
checking for stdlib.h... (cached) yes
checking for unistd.h... (cached) yes
checking for getpagesize... yes
checking for working mmap... yes
checking whether we are using the GNU C Library 2.1 or newer... yes
checking whether integer division by zero raises SIGFPE... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unsigned long long... yes
checking for inttypes.h... yes
checking whether the inttypes.h PRIxNN macros are broken... no
checking for ld used by GCC... ld
checking if the linker (ld) is GNU ld... yes
checking for shared library run path origin... done
checking argz.h usability... yes
checking argz.h presence... yes
checking for argz.h... yes
checking limits.h usability... yes
checking limits.h presence... yes
checking for limits.h... yes
checking locale.h usability... yes
checking locale.h presence... yes
checking for locale.h... yes
checking nl_types.h usability... yes
checking nl_types.h presence... yes
checking for nl_types.h... yes
checking malloc.h usability... yes
checking malloc.h presence... yes
checking for malloc.h... yes
checking stddef.h usability... yes
checking stddef.h presence... yes
checking for stddef.h... yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for unistd.h... (cached) yes
checking sys/param.h usability... yes
checking sys/param.h presence... yes
checking for sys/param.h... yes
checking for feof_unlocked... yes
checking for fgets_unlocked... yes
checking for getc_unlocked... yes
checking for getcwd... yes
checking for getegid... yes
checking for geteuid... yes
checking for getgid... yes
checking for getuid... yes
checking for mempcpy... yes
checking for munmap... yes
checking for putenv... yes
checking for setenv... yes
checking for setlocale... yes
checking for stpcpy... yes
checking for strcasecmp... yes
checking for strdup... yes
checking for strtoul... yes
checking for tsearch... yes
checking for __argz_count... yes
checking for __argz_stringify... yes
checking for __argz_next... yes
checking for __fsetlocking... yes
checking for iconv... yes
checking for iconv declaration... install-shextern size_t iconv (iconv_t cd, char * *inbuf, size_t *inbytesleft, char * *outbuf, size_t *outbytesleft);
checking for nl_langinfo and CODESET... yes
checking for LC_MESSAGES... yes
checking for bison... no
checking whether NLS is requested... yes
checking whether included gettext is requested... no
checking for GNU gettext in libc... yes
checking whether to use NLS... yes
checking where the gettext function comes from... libc
checking for aclocal... no
checking for autoconf... no
checking for autoheader... no
configure: updating cache ./config.cache
configure: creating ./config.status
config.status: creating Makefile
config.status: creating config.intl
config.status: creating config.h
config.status: executing default-1 commands
Configuring stage 1 in ./lto-plugin
configure: creating cache ./config.cache
checking build system type... armv7l-unknown-linux-gnu
checking host system type... armv7l-unknown-linux-gnu
checking target system type... armv7l-unknown-linux-gnu
checking for a BSD-compatible install... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking for style of include used by make... GNU
checking for armv7l-unknown-linux-gnu-gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking dependency style of gcc... gcc3
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/grep
checking for egrep... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking for armv7l-unknown-linux-gnu-gcc... (cached) gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to accept ISO C89... (cached) none needed
checking dependency style of gcc... (cached) gcc3
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... 64
checking whether gcc supports -Wall... yes
checking for -static-libgcc... yes
checking how to print strings... printf
checking for a sed that does not truncate output... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/sed
checking for fgrep... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/grep -F
checking for ld used by gcc... ld
checking if the linker (ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... nm
checking the name lister (nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking for ld option to reload object files... -r
checking for armv7l-unknown-linux-gnu-objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for armv7l-unknown-linux-gnu-ar... ar
checking for armv7l-unknown-linux-gnu-strip... strip
checking for armv7l-unknown-linux-gnu-ranlib... ranlib
checking command to parse nm output from gcc object... ok
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... no
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (ld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking for int64_t... yes
checking for uint64_t... yes
checking for sys/wait.h that is POSIX.1 compatible... yes
configure: updating cache ./config.cache
configure: creating ./config.status
config.status: creating Makefile
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing libtool commands
Configuring stage 1 in ./libiberty
configure: creating cache ./config.cache
checking whether to enable maintainer-specific portions of Makefiles... no
checking for makeinfo... makeinfo --split-size=5000000 --split-size=5000000
checking for perl... perl
checking build system type... armv7l-unknown-linux-gnu
checking host system type... armv7l-unknown-linux-gnu
checking for armv7l-unknown-linux-gnu-ar... ar
checking for armv7l-unknown-linux-gnu-ranlib... ranlib
checking whether to install libiberty headers and static library... no
configure: target_header_dir = 
checking for armv7l-unknown-linux-gnu-gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/grep
checking for egrep... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... 64
checking whether gcc supports -W... yes
checking whether gcc supports -Wall... yes
checking whether gcc supports -Wwrite-strings... yes
checking whether gcc supports -Wc++-compat... yes
checking whether gcc supports -Wstrict-prototypes... yes
checking whether gcc supports -Wshadow=local... no
checking whether gcc supports -pedantic ... yes
checking whether gcc and cc understand -c and -o together... yes
checking for an ANSI C-conforming const... yes
checking for inline... inline
checking whether byte ordering is bigendian... no
checking for a BSD-compatible install... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/install -c
checking for sys/file.h... yes
checking for sys/param.h... yes
checking for limits.h... yes
checking for stdlib.h... (cached) yes
checking for malloc.h... yes
checking for string.h... (cached) yes
checking for unistd.h... (cached) yes
checking for strings.h... (cached) yes
checking for sys/time.h... yes
checking for time.h... yes
checking for sys/resource.h... yes
checking for sys/stat.h... (cached) yes
checking for sys/mman.h... yes
checking for fcntl.h... yes
checking for alloca.h... yes
checking for sys/pstat.h... no
checking for sys/sysmp.h... no
checking for sys/sysinfo.h... yes
checking for machine/hal_sysinfo.h... no
checking for sys/table.h... no
checking for sys/sysctl.h... yes
checking for sys/systemcfg.h... no
checking for stdint.h... (cached) yes
checking for stdio_ext.h... yes
checking for process.h... no
checking for sys/prctl.h... yes
checking for sys/wait.h that is POSIX.1 compatible... yes
checking whether time.h and sys/time.h may both be included... yes
checking whether errno must be declared... no
checking size of int... 4
checking size of long... 4
checking size of size_t... 4
checking for long long... yes
checking size of long long... 8
checking for a 64-bit type... uint64_t
checking for intptr_t... yes
checking for uintptr_t... yes
checking for ssize_t... yes
checking for pid_t... yes
checking for library containing strerror... none required
checking for asprintf... yes
checking for atexit... yes
checking for basename... yes
checking for bcmp... yes
checking for bcopy... yes
checking for bsearch... yes
checking for bzero... yes
checking for calloc... yes
checking for clock... yes
checking for ffs... yes
checking for getcwd... yes
checking for getpagesize... yes
checking for gettimeofday... yes
checking for index... yes
checking for insque... yes
checking for memchr... yes
checking for memcmp... yes
checking for memcpy... yes
checking for memmem... yes
checking for memmove... yes
checking for mempcpy... yes
checking for memset... yes
checking for mkstemps... yes
checking for putenv... yes
checking for random... yes
checking for rename... yes
checking for rindex... yes
checking for setenv... yes
checking for snprintf... yes
checking for sigsetmask... yes
checking for stpcpy... yes
checking for stpncpy... yes
checking for strcasecmp... yes
checking for strchr... yes
checking for strdup... yes
checking for strncasecmp... yes
checking for strndup... yes
checking for strnlen... yes
checking for strrchr... yes
checking for strstr... yes
checking for strtod... yes
checking for strtol... yes
checking for strtoul... yes
checking for strtoll... yes
checking for strtoull... yes
checking for strverscmp... yes
checking for tmpnam... yes
checking for vasprintf... yes
checking for vfprintf... yes
checking for vprintf... yes
checking for vsnprintf... yes
checking for vsprintf... yes
checking for waitpid... yes
checking for setproctitle... no
checking whether alloca needs Cray hooks... no
checking stack direction for C alloca... -1
checking for vfork.h... no
checking for fork... yes
checking for vfork... yes
checking for working fork... yes
checking for working vfork... (cached) yes
checking for _doprnt... no
checking for sys_errlist... yes
checking for sys_nerr... yes
checking for sys_siglist... yes
checking for external symbol _system_configuration... no
checking for __fsetlocking... yes
checking for canonicalize_file_name... yes
checking for dup3... yes
checking for getrlimit... yes
checking for getrusage... yes
checking for getsysinfo... no
checking for gettimeofday... (cached) yes
checking for on_exit... yes
checking for psignal... yes
checking for pstat_getdynamic... no
checking for pstat_getstatic... no
checking for realpath... yes
checking for setrlimit... yes
checking for sbrk... yes
checking for spawnve... no
checking for spawnvpe... no
checking for strerror... yes
checking for strsignal... yes
checking for sysconf... yes
checking for sysctl... yes
checking for sysmp... no
checking for table... no
checking for times... yes
checking for wait3... yes
checking for wait4... yes
checking whether basename is declared... yes
checking whether ffs is declared... yes
checking whether asprintf is declared... yes
checking whether vasprintf is declared... yes
checking whether snprintf is declared... yes
checking whether vsnprintf is declared... yes
checking whether calloc is declared... yes
checking whether getenv is declared... yes
checking whether getopt is declared... yes
checking whether malloc is declared... yes
checking whether realloc is declared... yes
checking whether sbrk is declared... yes
checking whether strtol is declared... yes
checking whether strtoul is declared... yes
checking whether strtoll is declared... yes
checking whether strtoull is declared... yes
checking whether strverscmp is declared... yes
checking whether canonicalize_file_name must be declared... no
checking for stdlib.h... (cached) yes
checking for unistd.h... (cached) yes
checking for getpagesize... (cached) yes
checking for working mmap... yes
checking for working strncmp... yes
configure: updating cache ./config.cache
configure: creating ./config.status
config.status: creating Makefile
config.status: creating testsuite/Makefile
config.status: creating config.h
config.status: executing default commands
make[3]: Entering directory '/build/build/libiberty'
if [ x"-fPIC" != x ] && [ ! -d pic ]; then \
  mkdir pic; \
else true; fi
touch stamp-picdir
if [ x"" != x ] && [ ! -d noasan ]; then \
  mkdir noasan; \
else true; fi
touch stamp-noasandir
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/regex.c -o pic/regex.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/regex.c -o noasan/regex.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/regex.c -o regex.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/cplus-dem.c -o pic/cplus-dem.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/cplus-dem.c -o noasan/cplus-dem.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/cplus-dem.c -o cplus-dem.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/cp-demangle.c -o pic/cp-demangle.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/cp-demangle.c -o noasan/cp-demangle.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/cp-demangle.c -o cp-demangle.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/md5.c -o pic/md5.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/md5.c -o noasan/md5.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/md5.c -o md5.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/sha1.c -o pic/sha1.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/sha1.c -o noasan/sha1.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/sha1.c -o sha1.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/alloca.c -o pic/alloca.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/alloca.c -o noasan/alloca.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/alloca.c -o alloca.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/argv.c -o pic/argv.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/argv.c -o noasan/argv.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/argv.c -o argv.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/choose-temp.c -o pic/choose-temp.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/choose-temp.c -o noasan/choose-temp.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/choose-temp.c -o choose-temp.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/concat.c -o pic/concat.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/concat.c -o noasan/concat.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/concat.c -o concat.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/cp-demint.c -o pic/cp-demint.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/cp-demint.c -o noasan/cp-demint.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/cp-demint.c -o cp-demint.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/crc32.c -o pic/crc32.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/crc32.c -o noasan/crc32.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/crc32.c -o crc32.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/d-demangle.c -o pic/d-demangle.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/d-demangle.c -o noasan/d-demangle.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/d-demangle.c -o d-demangle.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/dwarfnames.c -o pic/dwarfnames.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/dwarfnames.c -o noasan/dwarfnames.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/dwarfnames.c -o dwarfnames.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/dyn-string.c -o pic/dyn-string.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/dyn-string.c -o noasan/dyn-string.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/dyn-string.c -o dyn-string.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/fdmatch.c -o pic/fdmatch.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/fdmatch.c -o noasan/fdmatch.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/fdmatch.c -o fdmatch.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/fibheap.c -o pic/fibheap.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/fibheap.c -o noasan/fibheap.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/fibheap.c -o fibheap.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/filename_cmp.c -o pic/filename_cmp.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/filename_cmp.c -o noasan/filename_cmp.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/filename_cmp.c -o filename_cmp.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/floatformat.c -o pic/floatformat.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/floatformat.c -o noasan/floatformat.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/floatformat.c -o floatformat.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/fnmatch.c -o pic/fnmatch.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/fnmatch.c -o noasan/fnmatch.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/fnmatch.c -o fnmatch.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/fopen_unlocked.c -o pic/fopen_unlocked.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/fopen_unlocked.c -o noasan/fopen_unlocked.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/fopen_unlocked.c -o fopen_unlocked.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/getopt.c -o pic/getopt.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/getopt.c -o noasan/getopt.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/getopt.c -o getopt.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/getopt1.c -o pic/getopt1.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/getopt1.c -o noasan/getopt1.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/getopt1.c -o getopt1.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/getpwd.c -o pic/getpwd.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/getpwd.c -o noasan/getpwd.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/getpwd.c -o getpwd.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/getruntime.c -o pic/getruntime.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/getruntime.c -o noasan/getruntime.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/getruntime.c -o getruntime.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/hashtab.c -o pic/hashtab.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/hashtab.c -o noasan/hashtab.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/hashtab.c -o hashtab.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/hex.c -o pic/hex.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/hex.c -o noasan/hex.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/hex.c -o hex.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/lbasename.c -o pic/lbasename.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/lbasename.c -o noasan/lbasename.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/lbasename.c -o lbasename.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/lrealpath.c -o pic/lrealpath.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/lrealpath.c -o noasan/lrealpath.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/lrealpath.c -o lrealpath.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/make-relative-prefix.c -o pic/make-relative-prefix.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/make-relative-prefix.c -o noasan/make-relative-prefix.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/make-relative-prefix.c -o make-relative-prefix.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/make-temp-file.c -o pic/make-temp-file.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/make-temp-file.c -o noasan/make-temp-file.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/make-temp-file.c -o make-temp-file.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/objalloc.c -o pic/objalloc.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/objalloc.c -o noasan/objalloc.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/objalloc.c -o objalloc.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/obstack.c -o pic/obstack.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/obstack.c -o noasan/obstack.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/obstack.c -o obstack.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/partition.c -o pic/partition.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/partition.c -o noasan/partition.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/partition.c -o partition.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/pexecute.c -o pic/pexecute.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/pexecute.c -o noasan/pexecute.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/pexecute.c -o pexecute.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/physmem.c -o pic/physmem.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/physmem.c -o noasan/physmem.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/physmem.c -o physmem.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/pex-common.c -o pic/pex-common.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/pex-common.c -o noasan/pex-common.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/pex-common.c -o pex-common.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/pex-one.c -o pic/pex-one.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/pex-one.c -o noasan/pex-one.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/pex-one.c -o pex-one.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/pex-unix.c -o pic/pex-unix.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/pex-unix.c -o noasan/pex-unix.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/pex-unix.c -o pex-unix.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/vprintf-support.c -o pic/vprintf-support.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/vprintf-support.c -o noasan/vprintf-support.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/vprintf-support.c -o vprintf-support.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/rust-demangle.c -o pic/rust-demangle.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/rust-demangle.c -o noasan/rust-demangle.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/rust-demangle.c -o rust-demangle.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/safe-ctype.c -o pic/safe-ctype.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/safe-ctype.c -o noasan/safe-ctype.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/safe-ctype.c -o safe-ctype.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/simple-object.c -o pic/simple-object.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/simple-object.c -o noasan/simple-object.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/simple-object.c -o simple-object.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/simple-object-coff.c -o pic/simple-object-coff.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/simple-object-coff.c -o noasan/simple-object-coff.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/simple-object-coff.c -o simple-object-coff.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/simple-object-elf.c -o pic/simple-object-elf.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/simple-object-elf.c -o noasan/simple-object-elf.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/simple-object-elf.c -o simple-object-elf.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/simple-object-mach-o.c -o pic/simple-object-mach-o.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/simple-object-mach-o.c -o noasan/simple-object-mach-o.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/simple-object-mach-o.c -o simple-object-mach-o.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/simple-object-xcoff.c -o pic/simple-object-xcoff.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/simple-object-xcoff.c -o noasan/simple-object-xcoff.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/simple-object-xcoff.c -o simple-object-xcoff.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/sort.c -o pic/sort.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/sort.c -o noasan/sort.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/sort.c -o sort.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/spaces.c -o pic/spaces.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/spaces.c -o noasan/spaces.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/spaces.c -o spaces.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/splay-tree.c -o pic/splay-tree.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/splay-tree.c -o noasan/splay-tree.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/splay-tree.c -o splay-tree.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/stack-limit.c -o pic/stack-limit.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/stack-limit.c -o noasan/stack-limit.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/stack-limit.c -o stack-limit.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/strerror.c -o pic/strerror.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/strerror.c -o noasan/strerror.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/strerror.c -o strerror.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/strsignal.c -o pic/strsignal.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/strsignal.c -o noasan/strsignal.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/strsignal.c -o strsignal.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/timeval-utils.c -o pic/timeval-utils.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/timeval-utils.c -o noasan/timeval-utils.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/timeval-utils.c -o timeval-utils.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/unlink-if-ordinary.c -o pic/unlink-if-ordinary.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/unlink-if-ordinary.c -o noasan/unlink-if-ordinary.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/unlink-if-ordinary.c -o unlink-if-ordinary.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/xasprintf.c -o pic/xasprintf.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/xasprintf.c -o noasan/xasprintf.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/xasprintf.c -o xasprintf.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/xatexit.c -o pic/xatexit.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/xatexit.c -o noasan/xatexit.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/xatexit.c -o xatexit.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/xexit.c -o pic/xexit.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/xexit.c -o noasan/xexit.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/xexit.c -o xexit.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/xmalloc.c -o pic/xmalloc.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/xmalloc.c -o noasan/xmalloc.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/xmalloc.c -o xmalloc.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/xmemdup.c -o pic/xmemdup.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/xmemdup.c -o noasan/xmemdup.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/xmemdup.c -o xmemdup.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/xstrdup.c -o pic/xstrdup.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/xstrdup.c -o noasan/xstrdup.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/xstrdup.c -o xstrdup.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/xstrerror.c -o pic/xstrerror.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/xstrerror.c -o noasan/xstrerror.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/xstrerror.c -o xstrerror.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/xstrndup.c -o pic/xstrndup.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/xstrndup.c -o noasan/xstrndup.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/xstrndup.c -o xstrndup.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/xvasprintf.c -o pic/xvasprintf.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/xvasprintf.c -o noasan/xvasprintf.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/xvasprintf.c -o xvasprintf.o
if [ x"-fPIC" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC ../../gcc-7.3.0/libiberty/setproctitle.c -o pic/setproctitle.o; \
else true; fi
if [ x"" != x ]; then \
  gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE -fPIC  ../../gcc-7.3.0/libiberty/setproctitle.c -o noasan/setproctitle.o; \
else true; fi
gcc -c -DHAVE_CONFIG_H -g  -I. -I../../gcc-7.3.0/libiberty/../include  -W -Wall -Wwrite-strings -Wc++-compat -Wstrict-prototypes -pedantic  -D_GNU_SOURCE ../../gcc-7.3.0/libiberty/setproctitle.c -o setproctitle.o
rm -f ./libiberty.a pic/./libiberty.a noasan/./libiberty.a
ar rc ./libiberty.a \
  ./regex.o ./cplus-dem.o ./cp-demangle.o ./md5.o ./sha1.o ./alloca.o ./argv.o ./choose-temp.o ./concat.o ./cp-demint.o ./crc32.o ./d-demangle.o ./dwarfnames.o ./dyn-string.o ./fdmatch.o ./fibheap.o ./filename_cmp.o ./floatformat.o ./fnmatch.o ./fopen_unlocked.o ./getopt.o ./getopt1.o ./getpwd.o ./getruntime.o ./hashtab.o ./hex.o ./lbasename.o ./lrealpath.o ./make-relative-prefix.o ./make-temp-file.o ./objalloc.o ./obstack.o ./partition.o ./pexecute.o ./physmem.o ./pex-common.o ./pex-one.o ./pex-unix.o ./vprintf-support.o ./rust-demangle.o ./safe-ctype.o ./simple-object.o ./simple-object-coff.o ./simple-object-elf.o ./simple-object-mach-o.o ./simple-object-xcoff.o ./sort.o ./spaces.o ./splay-tree.o ./stack-limit.o ./strerror.o ./strsignal.o ./timeval-utils.o ./unlink-if-ordinary.o ./xasprintf.o ./xatexit.o ./xexit.o ./xmalloc.o ./xmemdup.o ./xstrdup.o ./xstrerror.o ./xstrndup.o ./xvasprintf.o  ./setproctitle.o
ranlib ./libiberty.a
if [ x"-fPIC" != x ]; then \
  cd pic; \
  ar rc ./libiberty.a \
    ./regex.o ./cplus-dem.o ./cp-demangle.o ./md5.o ./sha1.o ./alloca.o ./argv.o ./choose-temp.o ./concat.o ./cp-demint.o ./crc32.o ./d-demangle.o ./dwarfnames.o ./dyn-string.o ./fdmatch.o ./fibheap.o ./filename_cmp.o ./floatformat.o ./fnmatch.o ./fopen_unlocked.o ./getopt.o ./getopt1.o ./getpwd.o ./getruntime.o ./hashtab.o ./hex.o ./lbasename.o ./lrealpath.o ./make-relative-prefix.o ./make-temp-file.o ./objalloc.o ./obstack.o ./partition.o ./pexecute.o ./physmem.o ./pex-common.o ./pex-one.o ./pex-unix.o ./vprintf-support.o ./rust-demangle.o ./safe-ctype.o ./simple-object.o ./simple-object-coff.o ./simple-object-elf.o ./simple-object-mach-o.o ./simple-object-xcoff.o ./sort.o ./spaces.o ./splay-tree.o ./stack-limit.o ./strerror.o ./strsignal.o ./timeval-utils.o ./unlink-if-ordinary.o ./xasprintf.o ./xatexit.o ./xexit.o ./xmalloc.o ./xmemdup.o ./xstrdup.o ./xstrerror.o ./xstrndup.o ./xvasprintf.o  ./setproctitle.o; \
  ranlib ./libiberty.a; \
  cd ..; \
else true; fi; \
if [ x"" != x ]; then \
  cd noasan; \
  ar rc ./libiberty.a \
    ./regex.o ./cplus-dem.o ./cp-demangle.o ./md5.o ./sha1.o ./alloca.o ./argv.o ./choose-temp.o ./concat.o ./cp-demint.o ./crc32.o ./d-demangle.o ./dwarfnames.o ./dyn-string.o ./fdmatch.o ./fibheap.o ./filename_cmp.o ./floatformat.o ./fnmatch.o ./fopen_unlocked.o ./getopt.o ./getopt1.o ./getpwd.o ./getruntime.o ./hashtab.o ./hex.o ./lbasename.o ./lrealpath.o ./make-relative-prefix.o ./make-temp-file.o ./objalloc.o ./obstack.o ./partition.o ./pexecute.o ./physmem.o ./pex-common.o ./pex-one.o ./pex-unix.o ./vprintf-support.o ./rust-demangle.o ./safe-ctype.o ./simple-object.o ./simple-object-coff.o ./simple-object-elf.o ./simple-object-mach-o.o ./simple-object-xcoff.o ./sort.o ./spaces.o ./splay-tree.o ./stack-limit.o ./strerror.o ./strsignal.o ./timeval-utils.o ./unlink-if-ordinary.o ./xasprintf.o ./xatexit.o ./xexit.o ./xmalloc.o ./xmemdup.o ./xstrdup.o ./xstrerror.o ./xstrndup.o ./xvasprintf.o  ./setproctitle.o; \
  ranlib ./libiberty.a; \
  cd ..; \
else true; fi
echo ./regex.o ./cplus-dem.o ./cp-demangle.o ./md5.o ./sha1.o ./alloca.o ./argv.o ./choose-temp.o ./concat.o ./cp-demint.o ./crc32.o ./d-demangle.o ./dwarfnames.o ./dyn-string.o ./fdmatch.o ./fibheap.o ./filename_cmp.o ./floatformat.o ./fnmatch.o ./fopen_unlocked.o ./getopt.o ./getopt1.o ./getpwd.o ./getruntime.o ./hashtab.o ./hex.o ./lbasename.o ./lrealpath.o ./make-relative-prefix.o ./make-temp-file.o ./objalloc.o ./obstack.o ./partition.o ./pexecute.o ./physmem.o ./pex-common.o ./pex-one.o ./pex-unix.o ./vprintf-support.o ./rust-demangle.o ./safe-ctype.o ./simple-object.o ./simple-object-coff.o ./simple-object-elf.o ./simple-object-mach-o.o ./simple-object-xcoff.o ./sort.o ./spaces.o ./splay-tree.o ./stack-limit.o ./strerror.o ./strsignal.o ./timeval-utils.o ./unlink-if-ordinary.o ./xasprintf.o ./xatexit.o ./xexit.o ./xmalloc.o ./xmemdup.o ./xstrdup.o ./xstrerror.o ./xstrndup.o ./xvasprintf.o > required-list
make[4]: Entering directory '/build/build/libiberty/testsuite'
make[4]: Nothing to be done for 'all'.
make[4]: Leaving directory '/build/build/libiberty/testsuite'
make[3]: Leaving directory '/build/build/libiberty'
make[3]: Entering directory '/build/build/lto-plugin'
make  all-am
make[4]: Entering directory '/build/build/lto-plugin'
/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/bash ./libtool --tag=CC --tag=disable-static  --mode=compile gcc -DHAVE_CONFIG_H -I. -I../../gcc-7.3.0/lto-plugin  -I../../gcc-7.3.0/lto-plugin/../include -DHAVE_CONFIG_H  -Wall -g -c -o lto-plugin.lo ../../gcc-7.3.0/lto-plugin/lto-plugin.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I../../gcc-7.3.0/lto-plugin -I../../gcc-7.3.0/lto-plugin/../include -DHAVE_CONFIG_H -Wall -g -c ../../gcc-7.3.0/lto-plugin/lto-plugin.c  -fPIC -DPIC -o .libs/lto-plugin.o
/nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/bash ./libtool --tag=CC --tag=disable-static  --mode=link gcc -Wall -g -Wc,-static-libgcc  -module -bindir /nix/store/3j52fq8qdrsfa60dn3ars8gk065bgpv0-gcc-7.3.0/libexec/gcc/armv7l-unknown-linux-gnu/7.3.0    -o liblto_plugin.la -rpath /nix/store/3j52fq8qdrsfa60dn3ars8gk065bgpv0-gcc-7.3.0/libexec/gcc/armv7l-unknown-linux-gnu/7.3.0 lto-plugin.lo  -Wc,../libiberty/pic/libiberty.a 
libtool: link: gcc -shared  -fPIC -DPIC  .libs/lto-plugin.o    -static-libgcc ../libiberty/pic/libiberty.a   -Wl,-soname -Wl,liblto_plugin.so.0 -o .libs/liblto_plugin.so.0.0.0
libtool: link: (cd ".libs" && rm -f "liblto_plugin.so.0" && ln -s "liblto_plugin.so.0.0.0" "liblto_plugin.so.0")
libtool: link: (cd ".libs" && rm -f "liblto_plugin.so" && ln -s "liblto_plugin.so.0.0.0" "liblto_plugin.so")
libtool: link: ( cd ".libs" && rm -f "liblto_plugin.la" && ln -s "../liblto_plugin.la" "liblto_plugin.la" )
libtool: install: /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/install -c .libs/liblto_plugin.so.0.0.0 /build/build/lto-plugin/../gcc/liblto_plugin.so.0.0.0
libtool: install: (cd /build/build/lto-plugin/../gcc && { ln -s -f liblto_plugin.so.0.0.0 liblto_plugin.so.0 || { rm -f liblto_plugin.so.0 && ln -s liblto_plugin.so.0.0.0 liblto_plugin.so.0; }; })
libtool: install: (cd /build/build/lto-plugin/../gcc && { ln -s -f liblto_plugin.so.0.0.0 liblto_plugin.so || { rm -f liblto_plugin.so && ln -s liblto_plugin.so.0.0.0 liblto_plugin.so; }; })
libtool: install: /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/install -c .libs/liblto_plugin.lai /build/build/lto-plugin/../gcc/liblto_plugin.la
libtool: install: warning: remember to run `libtool --finish /nix/store/3j52fq8qdrsfa60dn3ars8gk065bgpv0-gcc-7.3.0/libexec/gcc/armv7l-unknown-linux-gnu/7.3.0'
make[4]: Leaving directory '/build/build/lto-plugin'
make[3]: Leaving directory '/build/build/lto-plugin'
Configuring stage 1 in ./gcc
configure: creating cache ./config.cache
checking build system type... armv7l-unknown-linux-gnu
checking host system type... armv7l-unknown-linux-gnu
checking target system type... armv7l-unknown-linux-gnu
checking LIBRARY_PATH variable... ok
checking GCC_EXEC_PREFIX variable... ok
checking whether to place generated files in the source directory... no
checking whether a default linker was specified... no
checking whether a default assembler was specified... no
checking for armv7l-unknown-linux-gnu-gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables... 
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether we are using the GNU C++ compiler... yes
checking whether g++ -std=gnu++98 accepts -g... yes
checking for armv7l-unknown-linux-gnu-gnatbind... no
checking for armv7l-unknown-linux-gnu-gnatmake... no
checking whether compiler driver understands Ada... no
checking how to run the C++ preprocessor... g++ -std=gnu++98 -E
checking for grep that handles long lines and -e... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/grep
checking for egrep... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking how to run the C preprocessor... gcc -E
checking for inline... inline
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... 64
checking size of void *... 4
checking size of short... 2
checking size of int... 4
checking size of long... 4
checking for long long... yes
checking size of long long... 8
checking for int8_t... yes
checking for int16_t... yes
checking for int32_t... yes
checking for int64_t... yes
checking for long long int... yes
checking for intmax_t... yes
checking for intptr_t... yes
checking for uint8_t... yes
checking for uint16_t... yes
checking for uint32_t... yes
checking for uint64_t... yes
checking for unsigned long long int... yes
checking for uintmax_t... yes
checking for uintptr_t... yes
checking for int64_t underlying type... long long
checking for std::swap in <utility>... yes
checking whether g++ -std=gnu++98 is affected by placement new aliasing bug... no
checking whether g++ -std=gnu++98 supports -W... yes
checking whether g++ -std=gnu++98 supports -Wall... yes
checking whether g++ -std=gnu++98 supports -Wnarrowing... yes
checking whether g++ -std=gnu++98 supports -Wwrite-strings... yes
checking whether g++ -std=gnu++98 supports -Wcast-qual... yes
checking whether g++ -std=gnu++98 supports -Wformat... yes
checking whether gcc supports -Wstrict-prototypes... yes
checking whether gcc supports -Wmissing-prototypes... yes
checking whether g++ -std=gnu++98 supports -Wmissing-format-attribute... yes
checking whether g++ -std=gnu++98 supports -Woverloaded-virtual... yes
checking whether gcc supports -Wold-style-definition... yes
checking whether gcc supports -Wc++-compat... yes
checking whether g++ -std=gnu++98 supports -pedantic -Wlong-long -Wvariadic-macros -Woverlength-strings... yes
checking whether gcc supports -fno-exceptions... yes
checking whether gcc supports -fno-rtti... yes
checking whether gcc supports -fasynchronous-unwind-tables... yes
checking valgrind.h usability... no
checking valgrind.h presence... no
checking for valgrind.h... no
checking for VALGRIND_DISCARD in <valgrind/memcheck.h>... no
checking for VALGRIND_DISCARD in <memcheck.h>... no
checking for multiarch configuration... no, disabled auto check (configured with --native-system-header-dir)
configure: WARNING: decimal float is not supported for this target, ignored
checking whether make sets $(MAKE)... yes
checking for gawk... gawk
checking whether ln -s works... yes
checking whether ln works... yes
checking for armv7l-unknown-linux-gnu-ranlib... ranlib
checking for a BSD compatible install... /nix/store/lkqf5fk1q1vysrsn7dvywr6jhkhkq46s-bootstrap-tools/bin/install -c
checking for cmp's capabilities... gnucompare
checking for mktemp... yes
checking for makeinfo... makeinfo --split-size=5000000 --split-size=5000000
checking for modern makeinfo... yes
checking for recent Pod::Man... yes
checking for flex... /build/gcc-7.3.0/missing flex
checking for bison... /build/gcc-7.3.0/missing bison
checking for nm... nm
checking for ar... ar
checking for sphinx-build... texinfo
checking for ANSI C header files... (cached) yes
checking whether time.h and sys/time.h may both be included... yes
checking whether string.h and strings.h may both be included... yes
checking for sys/wait.h that is POSIX.1 compatible... yes
checking whether termios.h defines TIOCGWINSZ... no
checking whether sys/ioctl.h defines TIOCGWINSZ... yes
checking for limits.h... yes
checking for stddef.h... yes
checking for string.h... (cached) yes
checking for strings.h... (cached) yes
checking for stdlib.h... (cached) yes
checking for time.h... yes
checking for iconv.h... yes
checking for fcntl.h... yes
checking for ftw.h... yes
checking for unistd.h... (cached) yes
checking for sys/file.h... yes
checking for sys/time.h... yes
checking for sys/mman.h... yes
checking for sys/resource.h... yes
checking for sys/param.h... yes
checking for sys/times.h... yes
checking for sys/stat.h... (cached) yes
checking for direct.h... no
checking for malloc.h... yes
checking for langinfo.h... yes
checking for ldfcn.h... no
checking for locale.h... yes
checking for wchar.h... yes
checking for thread.h... no
checking for pthread.h... yes
checking for CHAR_BIT... yes
checking whether byte ordering is bigendian... no
checking how to run the C++ preprocessor... g++ -std=gnu++98 -E
checking for unordered_map... no
checking for tr1/unordered_map... yes
checking for ext/hash_map... yes
checking dependency style of g++ -std=gnu++98... gcc3
checking for collect2 libraries... none required
checking for library containing exc_resume... no
checking for library containing kstat_open... no
checking for library containing ldexp... none required
checking for inttypes.h... yes
checking for times... yes
checking for clock... yes
checking for kill... yes
checking for getrlimit... yes
checking for setrlimit... yes
checking for atoq... no
checking for popen... yes
checking for sysconf... yes
checking for strsignal... yes
checking for getrusage... yes
checking for nl_langinfo... yes
checking for gettimeofday... yes
checking for mbstowcs... yes
checking for wcswidth... yes
checking for mmap... yes
checking for setlocale... yes
checking for clearerr_unlocked... yes
checking for feof_unlocked... yes
checking for ferror_unlocked... yes
checking for fflush_unlocked... yes
checking for fgetc_unlocked... yes
checking for fgets_unlocked... yes
checking for fileno_unlocked... yes
checking for fprintf_unlocked... no
checking for fputc_unlocked... yes
checking for fputs_unlocked... yes
checking for fread_unlocked... yes
checking for fwrite_unlocked... yes
checking for getchar_unlocked... yes
checking for getc_unlocked... yes
checking for putchar_unlocked... yes
checking for putc_unlocked... yes
checking for madvise... yes
checking whether mbstowcs works... yes
checking for ssize_t... yes
checking for caddr_t... yes
checking for sys/mman.h... (cached) yes
checking for mmap... (cached) yes
checking whether read-only mmap of a plain file works... yes
checking whether mmap from /dev/zero works... yes
checking for MAP_ANON(YMOUS)... yes
checking whether mmap with MAP_ANON(YMOUS) works... yes
checking for pid_t... yes
checking for vfork.h... no
checking for fork... yes
checking for vfork... yes
checking for working fork... yes
checking for working vfork... (cached) yes
checking for ld used by GCC... ld
checking if the linker (ld) is GNU ld... yes
checking for shared library run path origin... done
checking for iconv... yes
checking for iconv declaration... install-shextern size_t iconv (iconv_t cd, char * *inbuf, size_t *inbytesleft, char * *outbuf, size_t *outbytesleft);
checking for LC_MESSAGES... yes
checking for nl_langinfo and CODESET... yes
checking whether basename is declared... yes
checking whether strstr is declared... yes
checking whether getenv is declared... yes
checking whether atol is declared... yes
checking whether atoll is declared... yes
checking whether asprintf is declared... yes
checking whether sbrk is declared... yes
checking whether abort is declared... yes
checking whether atof is declared... yes
checking whether getcwd is declared... yes
checking whether getwd is declared... yes
checking whether madvise is declared... yes
checking whether stpcpy is declared... yes
checking whether strnlen is declared... yes
checking whether strsignal is declared... yes
checking whether strverscmp is declared... yes
checking whether strtol is declared... yes
checking whether strtoul is declared... yes
checking whether strtoll is declared... yes
checking whether strtoull is declared... yes
checking whether setenv is declared... yes
checking whether unsetenv is declared... yes
checking whether errno is declared... yes
checking whether snprintf is declared... yes
checking whether vsnprintf is declared... yes
checking whether vasprintf is declared... yes
checking whether malloc is declared... yes
checking whether realloc is declared... yes
checking whether calloc is declared... yes
checking whether free is declared... yes
checking whether getopt is declared... yes
checking whether clock is declared... yes
checking whether getpagesize is declared... yes
checking whether ffs is declared... yes
checking whether clearerr_unlocked is declared... yes
checking whether feof_unlocked is declared... yes
checking whether ferror_unlocked is declared... yes
checking whether fflush_unlocked is declared... yes
checking whether fgetc_unlocked is declared... yes
checking whether fgets_unlocked is declared... yes
checking whether fileno_unlocked is declared... yes
checking whether fprintf_unlocked is declared... no
checking whether fputc_unlocked is declared... yes
checking whether fputs_unlocked is declared... yes
checking whether fread_unlocked is declared... yes
checking whether fwrite_unlocked is declared... yes
checking whether getchar_unlocked is declared... yes
checking whether getc_unlocked is declared... yes
checking whether putchar_unlocked is declared... yes
checking whether putc_unlocked is declared... yes
checking whether getrlimit is declared... yes
checking whether setrlimit is declared... yes
checking whether getrusage is declared... yes
checking whether ldgetname is declared... no
checking whether times is declared... yes
checking whether sigaltstack is declared... yes
checking for struct tms... yes
checking for clock_t... yes
checking for F_SETLKW... yes
checking if mkdir takes one argument... no
*** Configuration armv7l-unknown-linux-gnu not supported
make[2]: *** [Makefile:4274: configure-stage1-gcc] Error 1
make[2]: Leaving directory '/build/build'
make[1]: *** [Makefile:21249: stage1-bubble] Error 2
make[1]: Leaving directory '/build/build'
make: *** [Makefile:21570: bootstrap] Error 2
cannot build derivation '/nix/store/iis49fizzflqj7hi4v87rvdw3kryyx83-bootstrap-stage4-gcc-wrapper-7.3.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/xsn2587jpmr1i2xliskmaywhgkcdfr2m-bootstrap-stage4-stdenv-linux.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/8x6rlbibw5154ibgyqs4llfhs4f6zzmk-attr-2.4.47.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/hwbrcbxinffnaarph1ann9fnm60nma1y-bash-4.4-p19.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/gycxxwkzdb1mpng6qvkjjlhnrn3sj51m-binutils-2.30.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/230la4ma5dcdcvh6ind2v2zc485gb7hm-bzip2-1.0.6.0.1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/226kn5zl2vxnaxrj5nypm2ckfjdaha35-gawk-4.2.1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/pmp2wm8qks8j7xd9rwvxs7n96ghpdy6s-gnu-config-2016-12-31.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/wr93q0wxcbx8snpif3c2h3jv0lq16saz-gnumake-4.2.1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/ixsc8ijq1ipgqdqvg5l69wjc0b3gdl3z-gnused-4.4.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/56mh6ipcwabhwaxbp9kr5dkh2vg516qj-patchelf-0.9.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/2asxz17f2ga5yy02xs4sfbfdn1danfcq-paxctl-0.9.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/9kjcyn273nzr9kypxvbzxbkp6hk1dwvq-pcre-8.41.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/nms8kfwvkr9zzkycv2yf4788fvdzjnic-xz-5.2.3.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/vqqpz0yfgqacmp2l8kpkjbnqavv08z77-acl-2.2.52.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/z6y2xahm6sf0likyi5mbmjw6wvh2h8jl-gnugrep-3.1.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/pcgg65gs3l70al89p11pr1v8mkjz049p-gzip-1.9.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/488wym696ld2rylj7wiw3dvmqdqj3ybi-hook.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/2i6af99zyanqk7jyw8jca4sz6hgqbscb-texinfo-6.5.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/582wls0hgwdw23g144ak38i54jys4xvc-coreutils-8.29.drv': 4 dependencies couldn't be built
cannot build derivation '/nix/store/b45cwrilkcqvcz2wwc2zbi76n1g4ahzf-gnutar-1.30.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/kcd9jypvwdzi24sq85g8zady0dzkadyg-lzip-1.20.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/r27yz04qid20wysgz4bq5w3f74kfj13v-binutils-wrapper-2.30.drv': 4 dependencies couldn't be built
cannot build derivation '/nix/store/0wzjmgsgi339jrynh6iw5axn6glvk0l2-diffutils-3.6.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/lmh4njarq5mjcm4wzcsc39jxb9m7g86r-ed-1.14.2.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/6cbbsdnp5z5ixllqylahmnf3mshax52n-findutils-4.6.0.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/qf76yb3j7y7csylc2p1414jv2n0xg6zi-gcc-wrapper-7.3.0.drv': 5 dependencies couldn't be built
cannot build derivation '/nix/store/98jf8l82z0q4mfj3kgjfggz3hhq0vs39-patch-2.7.6.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/10g0yyvv1zvlb25qqzq37ig7ppjpw69g-stdenv-linux.drv': 25 dependencies couldn't be built
cannot build derivation '/nix/store/qipy7yy9z9jfpx0nfnjnycji1krgpmmv-hello-2.10.drv': 2 dependencies couldn't be built
error: build of '/nix/store/qipy7yy9z9jfpx0nfnjnycji1krgpmmv-hello-2.10.drv' failed
````

`config.guess` returns `armv7l-unknown-linux-gnueabihf`. Probably that's related.

Reply:
`armv7l-unknown-linux-gnueabihf` is recognized, and probably was already recognized, so a gcc patch is needed next.

Reply:
Well clearly it (and the several older GCCs that have been through this) doesn't need patches given it works fine you just don't give it these `--host` et al. in native builds and I don't know of any other distro patching their GCC for reasons like this.

Reply:
https://gcc.gnu.org/git/?p=gcc.git;a=commitdiff;h=8f50e6eddb95 is the culprit. Looks like they meant to blacklist odd-end arm vendors, so `unknown` is being caught by mistake.

Reply:
```diff
diff --git a/gcc/config.gcc b/gcc/config.gcc
index 7f5ba50d4b2..0acd953f119 100644
--- a/gcc/config.gcc
+++ b/gcc/config.gcc
@@ -254,6 +254,7 @@ esac
 case ${target} in
  # Avoid special cases that are not obsolete
    arm*-*-*eabi*                       \
+ | arm*-unknown-*                      \
  )
        ;;
    arm*-wince-pe*                      \
```

I'll submit

Reply:
Are you sure the real problem isn't that your quadruplet is missing the `*eabi*` part? What do other distros do?

Reply:
(There is an older 'OABI' ABI which has been dropped by GCC by now, I think).

Reply:
@dezgeg oh I see, interesting.